devops/scripts/update-pip-reqs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Needs to be run from repo root directory
```

Comment:
You can add this to ensure it's always run at the repo root. Portable across Linux/Mac.

```bash
get_script_dir() {
  declare src="${BASH_SOURCE[0]}"
  declare dir=

  while [ -h "$src" ]; do
    dir="$(cd -P "$( dirname "$src")" && pwd)"
    src="$(readlink "$src")"
    [[ $src != /* ]] && src="$dir/$src"
  done
  cd -P "$(dirname "$src")" && pwd 
}
cd "$(get_script_dir)/../../"
```